src/game.cpp

```diff
@@ -5742,6 +5758,13 @@ int game::assign_faction_id()
     return ret;
 }
 
+long game::assign_creature_id()
```

Comment:
On second thought creature tracker isn't serialized to the global game state, is it? I think it goes to the player save.  (not asking for this PR) We need to extract any game state that is world-specific to a separate class to shrink game and to make it explicit what that world state is to make resetting it when switching worlds clearer.  Until that happens I guess game is the right place for it after all.